xml/planning-planning-objectstorage-ring_specifications.xml

```diff
@@ -26,7 +26,7 @@
     <emphasis role="bold">Ring Specifications in &kw-hos; 2.x and 3.x</emphasis>
   </para>
   <para> In &kw-hos; 2.x and 3.x, ring specifications were mentioned in the
-      <literal>~/helion/my_cloud/definition/data/swift/rings.yml</literal> file. &kw-hos; 4.x
+      <literal>~/openstack/my_cloud/definition/data/swift/rings.yml</literal> file. &kw-hos; 4.x
```

Comment:
I think this content is probably outdated -- I can't imagine that this is still relevant to us. Can this section go?